File: JPPhotoManager/JPPhotoManager.Domain/SyncAssetsResult.cs
namespace JPPhotoManager.Domain
{
    public class SyncAssetsResult
    {
        public string SourceDirectory { get; set; }
        public string DestinationDirectory { get; set; }
        public int SyncedImages { get; set; }
        public string Message { get; set; }
    }
}

File: JPPhotoManagerWeb/backend/src/main/java/com/jpablodrexler/photomanager/application/dto/SyncAssetsResult.java
package com.jpablodrexler.photomanager.application.dto;

import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class SyncAssetsResult {

    private String sourceDirectory;
    private String destinationDirectory;
    private int syncedCount;
    private int deletedCount;
    private String message;
    private boolean success;

    public SyncAssetsResult(String sourceDirectory, String destinationDirectory) {
        this.sourceDirectory = sourceDirectory;
        this.destinationDirectory = destinationDirectory;
    }
}
